.pre-commit-config.yaml

```diff
@@ -22,8 +22,13 @@ repos:
         files: \.(md|rst|yml|yaml)
         args: [--prose-wrap=preserve]
 
-  - repo: https://github.com/asottile/pyupgrade
```

Comment:
Why remove pyupgrade?

Reply:
The pyupgrade rules are built into Ruff.

Reply:
It looks like it only implements some of pyupgrade, but I couldn't find a list of what it doesn't implement.

For example, how do you specify what the minimum version of Python you support is? I couldn't figure it out.

Reply:
Take a look at the pyproject.toml

@charliemarsh may be able to comment on which rules are in and out of scope.